### PR TITLE
Feature/#130 quote block

### PR DESCRIPTION
**Pull request by aleciafb**
https://projects.torchbox.com/projects/girl-effect-corporate-site/tickets/130

**Changes**
* style for one quote as per new designs
* style for quote block with no background
* quotation marks style 
* Font weight change **WIP**
* Gradient decreased
* Dropshadow toggle on/off dropshadow 
* Quote text color

### CODE REVIEW

File: girleffect/static_src/dist/css/main.css [hunk 1]
```diff
@@ -3417,12 +3418,11 @@ article .share-icons--hide-mobile {
...
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  flex-direction: row;
   margin: 0 auto;
-  align-items: center;
-  flex-wrap: wrap;
   align-items: stretch;
+  flex-wrap: wrap;
+  background-color: red;
```
Comment (nathanbegbie) [inline girleffect/static_src/dist/css/main.css]: @aleciafb I think this should be removed

File: girleffect/static_src/site/patterns/02-organisms/quote/quote.scss [hunk 2]
```diff
@@ -1,61 +1,58 @@
...
     margin: 0 auto;
-    align-items: center;
-    flex-wrap: wrap;
     align-items: stretch;
-
-    @include media-query(tablet-landscape) {
-        flex-direction: row;
-    }
+    flex-wrap: wrap;
+    background-color: red;
```
Comment (nathanbegbie) [inline girleffect/static_src/site/patterns/02-organisms/quote/quote.scss]: I think this is a test background color?

File: girleffect/static_src/dist/css/main.css [hunk 3]
```diff
@@ -3440,12 +3441,11 @@ article .share-icons--hide-mobile {
...
```
Comment (nathanbegbie) [inline girleffect/static_src/dist/css/main.css]: Test colour?

**Comment by nathanbegbie** [comment]
@aleciafb I think that the closing quote should be on the same line
screen shot 2018-01-16 at 17 20 20

**Comment by aleciafb** [comment]
@nathan That is the content pushing it to the next line. I'll have a look at what I can do

**Comment by nathanbegbie** [comment]
@aleciafb the third block doesn't seem to have the same height as the blocks above it.
screen shot 2018-01-16 at 17 28 54

This is the design below, which seems to have the same height for both.
screen shot 2018-01-16 at 17 29 03

**Comment by nathanbegbie** [comment]
The 3rd quote block does not seem to have the same responsive styles as the first and second blocks.
screen shot 2018-01-16 at 17 36 18

**Pull request closed without merging**